src/contracts/pods/EigenPodManager.sol

```diff
@@ -235,7 +239,7 @@ contract EigenPodManager is
 
     // INTERNAL FUNCTIONS
 
-    function _deployPod() internal onlyWhenNotPaused(PAUSED_NEW_EIGENPODS) returns (IEigenPod) {
+    function _deployPod() internal returns (IEigenPod) {
```

Comment:
did you move this to the external functions just for clarity or something?
seems these changes mildly change the behavior of the `stake` function -- it will now revert if `PAUSED_NEW_EIGENPODS` is set, even if the caller already has a deployed EigenPod.
I don't care strongly either way, but think I have a slight preference for the previous behavior.

Reply:
no, i moved it so that we could pause calls to `EigenPod.stake` in all scenarios - since that's a potentially dangerous state changing method.

I want to ensure that if we find out something is wrong with `stake` and interaction with the deposit contract that we can prevent its use temporarily. IMO this is much more important than preventing the deployment of pods specifically, since that's a really simple process

Reply:
works for me 👍